dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -500,6 +500,7 @@ void ExchangeReceiverBase<RPCContext>::setUpConnection()
         });
 
         ++thread_count;
+        --connection_uncreated_num;
```

Comment:
It looks like `connection_uncreated_num-=async_requests.size()` is the correct one?

Reply:
> It looks like `connection_uncreated_num-=async_requests.size()` is the correct one?

I will fix it